convert_hcl_to_json.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+install_jq(){
+    OS=$(uname -s)
+    if [[ "$OS" == "Darwin" ]]; then
+        OS="macos"
+    elif [[ "$OS" == "Linux" ]]; then
+        OS="linux"
+    else
+        echo "Unsupported OS: $OS"
+        exit 1
+    fi
+
+    ARCH=$(uname -m)
+
+    JQ_BIN="/tmp/jq"
+    url="https://github.com/jqlang/jq/releases/download/jq-1.8.1/jq-${OS}-${ARCH}"
+    curl -L -o $JQ_BIN $url
+    chmod +x $JQ_BIN
+
+}
+
+install_hcl2json(){
+    OS=$(uname -s)
+    if [[ "$OS" == "Darwin" ]]; then
+        OS="darwin"
+    elif [[ "$OS" == "Linux" ]]; then
+        OS="linux"
+    else
+        echo "Unsupported OS: $OS"
+        exit 1
+    fi
+
+    ARCH=$(uname -m)
+
+    HCL2JSON_BIN="/tmp/hcl2json"
+
+    url="https://github.com/tmccombs/hcl2json/releases/download/v0.6.7/hcl2json_${OS}_${ARCH}"
+    echo $url
+    curl -L -o $HCL2JSON_BIN $url
+    chmod +x $HCL2JSON_BIN
+}
+
+
+INPUT_FILE_JSON="$1"
+if [ -z "$INPUT_FILE_JSON" ]; then
```

Comment:
Let's switch from single `[ ]` to `[[ ]]` since we are already writing everything in `bash`.

Also, if we require file as input, check with:
```
if [[ -f "$INPUT_FILE_JSON" ]]; then
  ...
fi